docs/modules/record-encryption/proc-configuring-record-encryption-filter.adoc

```diff
@@ -35,14 +35,20 @@ filters:
       selector: <KEK_selector_service_name> # <3>
       selectorConfig:
         template: "KEK_${topicName}" # <4>
+      experimental: # <5>
+        "encryptionDekExpireAfterWriteSeconds": 7200 # <6>
+        "maxEncryptionsPerDek": 5000000 # <7>
 	 # ...
 ----
 <1> The KMS service name.
 <2> Configuration specific to the KMS provider.
 <3> The Key Encryption Key (KEK) selector to use. The `${topicName}` is a literal understood by the proxy. 
 For example, if using the `TemplateKekSelector` with the template `KEK_$\{topicName}`, create a key for every topic that
 is to be encrypted with the key name matching the topic name, prefixed by the string `KEK_`.
-<4> The template for deriving the KEK, based on a specific topic name. 
+<4> The template for deriving the KEK, based on a specific topic name.
+<5> **Warning**: experimental configurations do not offer compatibility guarantees
+<6> How long to use a DEK for encryption after it has been created, this can be used to control how quickly we use new key material after an external key rotation of the KEK. Note that decreasing this value has a cost in increased operations on the KMS at encrypt and decrypt time, because more DEKs will be created and used.
```

Comment:
```suggestion
<6> How long to use a DEK for encryption after it has been created, this can be used to control how quickly we respond to the fact that a KEK has been rotated externally. Note that tuning this value has a cost implications. Shorter durations will result in increased operations on the KMS at encrypt and decrypt time, because more DEKs will be created and used.
```

Reply:
incorporated these fixes with a little grammar fix, I've also juggled this description to describe the refresh setting as it's more likely to be the setting that affects an actively used DEK